ldscript.txt

```diff
@@ -370,7 +374,10 @@ SECTIONS
         build/sa2/src/sprite_2.o(.rodata);
         build/sa2/src/animation_commands.o(.rodata);
         build/sa2/src/sprite_3.o(.rodata);
-        build/sa2/asm/sprite_4.o(.rodata);
+        build/sa2/src/bg_triangles.o(.rodata);
+        build/sa2/src/malloc_ewram.o(.rodata);
+        build/sa2/src/malloc_vram.o(.rodata);
+        build/sa2/src/input_recorder.o(.rodata);
```

Comment:
So there is no rodata here, I guess you added to make modification easier? I wonder if this is needed

Reply:
Yeah, basically so people who aren't familiar with the linker script stuff can edit the files without any linker errors of the ".rodata referenced, etc. etc." kind appearing. ^^